spec/services/idv/agent_spec.rb

```diff
@@ -53,88 +45,71 @@
       end
     end
 
-    describe '#proof' do
-      let(:resolution_message) { 'reason 1' }
-      let(:state_id_message) { 'reason 2' }
-      let(:failed_message) { 'bah humbug' }
-      let(:error) { { bad: 'stuff' } }
-      let(:exception) { StandardError.new }
-
-      subject { agent.proof(*stages) }
-
-      before do
-        exception_instance = exception
-        allow(Idv::Proofer).to receive(:get_vendor) do |stage|
-          logic = case stage
-                  when :resolution
-                    proc { |_, r| r.add_message('reason 1') }
-                  when :state_id
-                    proc { |_, r| r.add_message('reason 2') }
-                  when :failed
-                    proc { |_, r| r.add_message('bah humbug').add_error(:bad, 'stuff') }
-                  when :timed_out
-                    proc { |_, r| r.instance_variable_set(:@exception, Proofer::TimeoutError.new) }
-                  when :exception
-                    proc { |_, r| r.instance_variable_set(:@exception, exception_instance) }
-                  end
-          Class.new(Proofer::Base) do
-            required_attributes(:foo)
-            proof(&logic)
-          end
+    describe '#proof_resolution' do
+      context 'proofing state_id enabled' do
+        it 'does not proof state_id if resolution fails' do
+          agent = Idv::Agent.new({ ssn: '444-55-6666', first_name: Faker::Name.first_name,
```

Comment:
nit: I'd drop the curly braces here, since the last argument in Ruby doesn't need them, ditto for the rest of the file